### PR TITLE
helm_info: add release_state argument

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Specify release state in helm list command as per helm cmdline flags.

Fixes: #377

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
changelogs/fragments/377-helm-info-state.yml
molecule/default/roles/helm/tasks/tests_chart.yml
plugins/modules/helm_info.py

### CODE REVIEW

File: plugins/modules/helm_info.py [hunk 1]
```diff
@@ -39,15 +39,35 @@
     required: true
     type: str
     aliases: [ namespace ]
+  release_state:
+    description:
+      - Show releases as per their states.
+      - Default value is C(deployed).
```
[the remark this answers is not in the document]
Comment (Akasurde) [reply plugins/modules/helm_info.py:45]: `deployed` is by default in the command itself. It does not matter you add the command flag or not.

**Comment by Akasurde** [comment]
recheck

**Review by abikouo — APPROVED** [review]

**Comment by Akasurde** [comment]
recheck

**Review by gravesm — COMMENTED** [review]
Small doc change, but otherwise lgtm. Thanks!

**Comment by Akasurde** [comment]
recheck

**Pull request merged**